ros_ws/src/navigation_stack/msgs_converter/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(msgs_converter)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+set (CMAKE_CXX_STANDARD 11)
+
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+  drive_msgs
+  roscpp
+  rospy
+  std_msgs
+  vesc_msgs
+  nav_msgs
+  vesc_sim
+)
+
+
```

Comment:
Please add the line that sets c++ warnings from one of the other CMake files, such as `ros_ws/src/navigation_stack/msgs_converter/CMakeLists.txt`.